cmd/agent/main.go

```diff
@@ -358,6 +360,7 @@ func parseFlags() *options {
 }
 
 func setupLogging(ctx context.Context, o *options) {
+	log.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
I know logging isn't great, but could you keep it as is for now?

FYI, I have a draft to improve it: https://github.com/admiraltyio/admiralty/pull/214

Reply:
removed